doc/developers/contributing.rst

```diff
@@ -641,22 +641,42 @@ the corresponding subpackages.
 
 We expect code coverage of new features to be at least around 90%.
 
-.. note:: **Workflow to improve test coverage**
+For guidelines on how to use ``pytest`` efficiently, see the
+:ref:`pytest_tips`.
 
-   To test code coverage, you need to install the `coverage
-   <https://pypi.org/project/coverage/>`_ package in addition to pytest.
+Available test fixtures in scikit-learn
+.......................................
 
-   1. Run 'make test-coverage'. The output lists for each file the line
-      numbers that are not tested.
+Test fixtures ensure that a set of tests will be executing with the appropriate
+initialization and cleanup. The scikit-learn test suite implement the following
+fixtures which should be used whenever the testing conditions match:
 
-   2. Find a low hanging fruit, looking at which lines are not tested,
-      write or adapt a test specifically for these lines.
+`plt`
+    The `plt` fixture should be used when a test function is dealing with
+    `matplotlib`. `matplotlib` is a soft dependency and is not required. This
+    fixture is charge of skipping the tests if `matplotlib` is not installed.
+    In addition, figures created during the tests will be automatically closed
+    once the test function has been executed.
 
-   3. Loop.
+To use one of these fixtures in a test function, one needs to pass it as an
```

Comment:
We only have one fixture documented here, and yet the whole section suggests there are more than one. It's not that confusing, but it's a bit strange.